python/publish/__init__.py

```diff
@@ -786,7 +787,7 @@ def get_case_annotation(messages: CaseMessages,
                                        for m in messages[key][state]
                                        for c in messages[key][state][m]])
                          if case.result_file}
-    test_file = case.test_file
+    test_file = case.test_file.replace(test_file_path_prefix, "", 1)
```

Comment:
This would also remove the prefix from the test file if it does not occur as a prefix, Not very likely but better be safe:
```suggestion
    test_file = case.test_file[len(test_path_file_prefix):] if test_path_file_prefix and case.test_file.startswith(test_path_file_prefix) else case.test_file
```